drizzle/doblot.py

```diff
@@ -69,7 +69,7 @@ def doblot(source, source_wcs, blot_wcs, exptime, coeffs = True,
         Was used when input to output mapping was computed
         internally. Is no longer used and only here for backwards compatibility.
     """
-    _outsci = np.zeros((blot_wcs._naxis2,blot_wcs._naxis1),dtype=np.float32)
+    _outsci = np.zeros(blot_wcs.pixel_shape[::-1],dtype=np.float32)
```

Comment:
@jamienoss, any chance you could make a small change to this PR, such as putting a space before `dtype` here?  That will kick of the CI tests and maybe we can get this merged.  Thanks!  😄 

In case you haven't been following the saga, the Travis CI tests on this repo have been broken for, um, a while.  Now they're back.

Reply:
Thanks!  Tests pass.  Merging.